ntoskrnl/include/internal/i386/intrin_i.h

```diff
@@ -5,6 +5,45 @@ extern "C"
 {
 #endif
 
+FORCEINLINE
+PKGDTENTRY
+KiGetGdtEntry(
+    IN PVOID pGdt,
+    IN USHORT Selector)
+{
+    return (PKGDTENTRY)((ULONG_PTR)pGdt + (Selector & ~RPL_MASK));
+}
+
+FORCEINLINE
+VOID
+KiSetGdtDescriptorBase(
+    IN OUT PKGDTENTRY Entry,
+    IN UINT32 Base)
+{
+    Entry->BaseLow = (UINT16)(Base & 0xffff);
```

Comment:
What's the advantage of having changed the original USHORT / ULONG etc. used in these functions, into UINT16/UNT32 etc?

Reply:
Any answer there?

Reply:
i changed it back, grabbing what was in freeloader as is.